### ISSUE TITLE
train_nlp.py中logits的疑惑

### ISSUE THREAD

**Issue by LanCole** [WENGSYX/CirBERTa#3]
output = model(input_ids=input_ids, input_mask=attention_mask,  labels=y)
            logits = output['logits']
            logits[:,:64243]=0
            logits[:,64244:68158]=0
            logits[:,68159:110093]=0
            logits[:,110094:115270]=0
            logits[:,115271:125883]=0
            logits[:,125884:]=0
            loss = criterion(logits,output['labels'])

为什么要对logits[:,:]赋值为0

**Comment by LanCole** [comment]
最后logits全为0了，loss值怎么计算呢

**Comment by WENGSYX** [comment]
没有全为0，这不是还剩下五个嘛，这五个就是用来分类的，其他的logits归零，为了让模型更关注这五个Token
